assesspy/utils.py

```diff
@@ -1,35 +1,30 @@
-# Import necessary libraries
 import numpy as np
 import pandas as pd
 from pandas.api.types import is_numeric_dtype
 
 
-def check_inputs(*args):
-    out = [""]
-
-    for x in args:
-        # *args passed into *args can created nested tuples - unnest
-        if isinstance(x, tuple):
-            args = x
-
+def check_inputs(*args, check_gt_zero: bool = True) -> None:
+    out_msg = [""]
     for x in args:
-        if isinstance(x, pd.core.frame.DataFrame):
-            raise Exception("Input cannot be a dataframe.")
-
         check = pd.Series(x)
 
         if not is_numeric_dtype(check):
-            raise Exception("All input vectors must be numeric.")
-        if check.isnull().values.any():
-            out.append("\nInput vectors contain null values.")
+            out_msg.append("\nAll input values must be numeric.")
```

Comment:
**[Nitpick, non-blocking]** Instead of prepending each error string with a newline, could we just add newlines as separators during the `join()`?